lib/list.gd

```diff
@@ -2376,35 +2376,19 @@ DeclareGlobalFunction( "ListWithIdenticalEntries" );
 ##
 #F  PlainListCopy( <list> ) . . . . . . . .  make a plain list copy of a list
 ##
+##  <#GAPDoc Label="PlainListCopy">
 ##  <ManSection>
 ##  <Func Name="PlainListCopy" Arg='list'/>
 ##
 ##  <Description>
+##  This function returns a list equal to its argument, in a plain list
+##  representation (see <Ref Filt="IsPlistRep"/>).
 ##  This is intended for use in certain rare situations,
-##  such as before objectifying.
-##  Normally, <C>ConstantAccessTimeList</C> should be enough.
-##  </Description>
-##  </ManSection>
-##
-DeclareGlobalFunction("PlainListCopy");
-
-
-#############################################################################
-##
-#O  PlainListCopyOp( <list> ) . . . . . . . .return a plain version of a list
-##
-##  <ManSection>
-##  <Oper Name="PlainListCopyOp" Arg='list'/>
-##
-##  <Description>
-##  This operation returns a list equal to its argument, in a plain list
-##  representation. This may be the argument converted in place, or
-##  may be new. It is only intended to be called by <C>PlainListCopy</C>.
+##  such as before objectifying, or calling some kernel functions.
 ##  </Description>
 ##  </ManSection>
+##  <#/GAPDoc>
 ##
-DeclareOperation("PlainListCopyOp", [IsSmallList]);
```

Comment:
Should we say here that it is a kernel function, in case people wonder where to find it?

Reply:
(I am also fine with merging as-is, just wondering)

Reply:
I've now moved the documentation inline, because it was weird here -- but maybe that's a bad idea too. We could start scanning C code for GAPDoc?

Reply:
In `doc/ref/makedocreldata.g` we already list `src/sysfiles.c` and could add more. I would in fact suggest that we change that file to not hardcode the list of files to scan, but rather to scan all files `src/*.{c,h}`, `lib/*.{gd,gi}`, etc., but of course that goes way beyond the scope of this PR ;-)